bin/hdfcoinc/pycbc_multiifo_coinc_statmap

```diff
@@ -0,0 +1,472 @@
+#!/usr/bin/env python
```

Comment:
Its kind of hard to look over this if it is mostly a copy of the non-multi-ifo approved code. We should probably see a diff of this with respect to the original code. Could that be linked from the issue tracker?

Reply:
Do you mean, that I should create a diff of the two files, and put it into an issue? I will do that over the weekend as well.

Reply:
I added the diff into a new issue: https://github.com/gwastro/pycbc/issues/2452